### PR TITLE
Release notes for v1.6.0

### PR DESCRIPTION
**Pull request by danielballan**
This assumes #931 will be merged.

### CODE REVIEW

**Comment by danielballan** [comment]
@prjemian In particular, you would confirm that my description of the changes/additions/fixes related to `write_timeout` and `auto_monitor` are correct? I can easily imagine referring back to these release notes to understand those changes in the future.

**Review by prjemian — APPROVED** [review]
LGTM

File: doc/source/release_notes.rst [hunk 1]
```diff
@@ -2,13 +2,58 @@
...
+  Some log messages have been moved from ``'ophyd.object'`` to
+  ``'ophyd.control_layer'``, and the level of messages emitted to
+  ``'ophyd.control_layer'`` have been reduced from INFO to DEBUG. Some log
+  messages have been removed entirely.
+* The method :meth:`EpicsSignalBase.set_default_timeout` has been renamed to
+  :meth:`EpicsSignalBase.set_defaults` because it has been extended to include
+  more than timeouts, as described above. The old name is still supported but
+  issues a warning that it may be removed in the future.
+* Use auto-monitoring in more places in ``EpicsMotor``. This should drastically
+  speed up ``motor.read_conifiguration()``.
```
Comment (prjemian) [inline doc/source/release_notes.rst:48]: typo:  ``motor.read_configuration()``

File: interactive_test.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+RE = RunEngine()
+db = temp()
+catalog = db.v2
+RE.subscribe(db.insert)
+
+PV_PREFIX = "13SIM1:"
+# prefixes = ['13SIM1:', 'XF:31IDA-BI{Cam:Tbl}']
+
+# Create a directory for the HDF5 file which we will cleanup at the end.
```
Comment (prjemian) [inline interactive_test.py:23]: Where's the cleanup?

**Review by prjemian — DISMISSED** [review]
1. fix typo
1. question (as noted)

File: interactive_test.py (hunk 2 above)
Comment (danielballan) [reply interactive_test.py]: This file was accidentally committed. Removed and force-pushed.

File: doc/source/release_notes.rst [hunk 3]
```diff
@@ -2,13 +2,58 @@
...
-===================
+1.6.0 (2020-11-09)
+==================
 
-Added
+Fixes
 -----
 
+* The ``write_timeout`` specified for an :class:`EpicsSignalBase` is respected
+  by its ``set()`` method unless overridden with ``set(.., timeout=TIMEOUT)``.
```
Comment (mrakitin) [inline doc/source/release_notes.rst:12]:
```suggestion
  by its ``set()`` method unless overridden with ``set(..., timeout=TIMEOUT)``.
```

File: doc/source/release_notes.rst [hunk 4]
```diff
@@ -2,13 +2,58 @@
  Release History
 =================
 
-v1.6.0 (2020-XX-XX)
-===================
+1.6.0 (2020-11-09)
```
Comment (mrakitin) [inline doc/source/release_notes.rst:5]:
```suggestion
1.6.0 (2020-11-12)
```

File: doc/source/release_notes.rst (hunk 1 above)
Comment (mrakitin) [inline doc/source/release_notes.rst:48]: Seconding @prjemian's suggestion.
```suggestion
  speed up ``motor.read_configuration()``.
```

**Review by mrakitin — COMMENTED** [review]
Looks good. Suggested a few updates.

**Comment by danielballan** [comment]
One of the three tests had an intermittent failure I have seen a couple times. Tracking that here: https://github.com/bluesky/ophyd/issues/956

I don't think it should block merging or release.

File: docs/source/release_notes.rst [hunk 5]
```diff
@@ -2,13 +2,66 @@
...
+  by its ``set()`` method unless overridden with ``set(..., timeout=TIMEOUT)``.
+* Area Detector PVs related to array shape have been given an order compatible
+  with the numpy array index ordering of the array itself.
+* Thread the keyword ``EpicsSignal.get(..., use_monitor=True)`` down to the
+  control system. This setting was previously supported but support was removed
+  (years ago). We now view its removal as a mistake.
+* When area detector takes a series of images and ``num_capture`` is set to
+  ``0``, this is now interpreted to mean "however many images the detector is
+  configured to acquire" rather than "0 images".
+* Fixed EigerDetectorCam's photon_energy component's PV suffix spelling.
```
Comment (mrakitin) [inline docs/source/release_notes.rst:21]:
```suggestion
* Fixed EigerDetectorCam's ``photon_energy`` component's PV suffix spelling.
```

**Review by mrakitin — APPROVED** [review]
Looks good to me! I added a single, non-blocking suggestion. Feel free to self-merge.

**Comment by danielballan** [comment]
Your detailed reviews are always appreciated, @mrakitin.

**Pull request merged**